packages/core/deploy/036_deploy_across_ovm_bridge_deposit_box.js

```diff
@@ -0,0 +1,30 @@
+// This deploy script should be run on an optimism provider.
+const { getAddress } = require("@uma/contracts-node");
```

Comment:
We might want to be cautious about depending on contracts-node in core since it creates a circular dep. Is there a way to use hardhat directly to grab the admin address (there might not be, and we might be able to just leave this as is and get away with it)?

Reply:
I'm not really sure what the best pattern is for this and you do raise an excellent point.

More generally, this kind of migration does not follow the patterns our other ones do as it returns early if not running on specific chain IDs. it might be a good idea to put this in a different directory `ovm_deploy` or something to this effect that does not run as part of the normal migration and gives us the space to separate out these kinds of migrations. wdyt?